src/diff.js

```diff
@@ -41,8 +41,8 @@ export function render(diff) {
   }
 }
 
-export default function diffLogger(prevState, newState, logger, isCollapsed) {
-  const diff = differ(prevState, newState);
+export default function diffLogger(prevState, newState, logger, isCollapsed, customDiffer) {
+  const diff = (customDiffer || differ)(prevState, newState);
```

Comment:
Maybe you could use [default paremeters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters).